apis/core/v1alpha1/commonpackage_types.go

```diff
@@ -67,3 +67,14 @@ type PackageSpec struct {
 	// If Paused is true, the package and its children will not be reconciled.
 	Paused bool `json:"paused,omitempty"`
 }
+
+// PackageTemplateSpec describes the data a package should have when created from a template.
+type PackageTemplateSpec struct {
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	// Specification of the desired behavior of the package.
+	// +optional
+	Spec PackageSpec `json:"spec,omitempty"`
```

Comment:
I don't think this field should be optional.
```suggestion
	// Specification of the desired behavior of the package.
	Spec PackageSpec `json:"spec"`
```

Reply:
fixed.